### ISSUE TITLE
example is not using defined user

### ISSUE THREAD

**Issue by martinadolfi** [docker/docker.github.io#2748]
File: [compose/wordpress.md](https://docs.docker.com/compose/wordpress/), CC @londoncalling 
I've been testing this example, and this compose example makes wordpress log in to mysql using the root user instead of the "wordpress" user defined in the database.... I'm almost sure ;)

**Comment by londoncalling** [comment]
### Hackathon Hints

Please test the code sample for @martinadolfi 's comment, and if correct, identify how to update the sample. If it's something you can change simply by changing how it's documented, then do so. Otherwise, suggest a change to the code if possible.

**Comment by martinadolfi** [comment]
Just to help, I think the easiest way to thest this is by changing root password like this (i added an `x` to the password) ,and then starting with `docker-compose up`:
```yml
version: '2'

services:
   db:
     image: mysql:5.7
     volumes:
       - db_data:/var/lib/mysql
     restart: always
     environment:
       MYSQL_ROOT_PASSWORD: wordprex
       MYSQL_DATABASE: wordpress
       MYSQL_USER: wordpress
       MYSQL_PASSWORD: wordpress

   wordpress:
     depends_on:
       - db
     image: wordpress:latest
     ports:
       - "8000:80"
     restart: always
     environment:
       WORDPRESS_DB_HOST: db:3306
       WORDPRESS_DB_PASSWORD: wordpress
volumes:
    db_data:
```

You´ll see this i the log:

> wordpress_1  | Warning: mysqli::mysqli(): (HY000/1045): Access denied for user 'root'@'172.22.0.3' (using password: YES) in - on line 22
db_1         | 2017-04-13T13:52:21.837829Z 4 [Note] Access denied for user 'root'@'172.22.0.3' (using password: YES)

So i'm pretty sure it's trying to access with root.

**Comment by bhavin192** [comment]
You are right @martinadolfi, as it defaults to root if not specified. It tries to connect to database with root password. Good catch 👍

**Comment by mstanleyjones** [comment]
Closed via #2824
